### PR TITLE
gtkui: add hotkey for track properties

### LINKED ISSUE #1522: keyboard shortcut for accessing track properties window
in fb2k it is alt+enter

### PR DESCRIPTION
**Pull request by kuba160**
Use Alt+Return hotkey to open Track properties by default. This closes #1522 .

### CODE REVIEW

**Comment by Alexey-Yakovenko** [comment]
waaait.. why there are these duplicate lines?

File: plugins/gtkui/hotkeys.c [hunk 1]
```diff
@@ -919,7 +919,7 @@ gtkui_set_default_hotkeys (void) {
     deadbeef->conf_set_str ("hotkey.key10", "\"Delete\" 1 0 remove_from_playlist");
     deadbeef->conf_set_str ("hotkey.key11", "\"Ctrl w\" 0 0 remove_current_playlist");
     deadbeef->conf_set_str ("hotkey.key11", "\"Ctrl w\" 0 0 remove_current_playlist");
```
Comment (Alexey-Yakovenko) [inline plugins/gtkui/hotkeys.c]: Since you're at this already, could you please remove the other duplicate as well?

Comment (kuba160) [inline plugins/gtkui/hotkeys.c]: I thought it's made to make space so they can be used in the future.

Comment (kuba160) [inline plugins/gtkui/hotkeys.c]: Should I remove whole line or just leave the line blank?

Comment (Alexey-Yakovenko) [inline plugins/gtkui/hotkeys.c]: Remove the line please.

**Pull request merged**